access/ms-teams/bot.go

```diff
@@ -125,52 +136,72 @@ func (b Bot) UninstallAppForUser(ctx context.Context, userIDOrEmail string) erro
 	return trace.Wrap(err)
 }
 
-// FetchUser fetches app id for user, installs app for a user if missing, fetches chat id and saves
-// everything to cache. This method is used for priming the cache. Returns trace.NotFound if a
+// FetchRecipient checks if recipient is a user or a channel, installs app for a user if missing, fetches chat id
+// and saves everything to cache. This method is used for priming the cache. Returns trace.NotFound if a
 // user was not found.
-func (b Bot) FetchUser(ctx context.Context, userIDOrEmail string) (*UserData, error) {
+func (b Bot) FetchRecipient(ctx context.Context, recipient string) (*RecipientData, error) {
```

Comment:
Umm.... Perhaps a silly question, but is there any reason why we're using Value receivers on these methods?

I mean, yes, it works OK because `mu` is a pointer to a `*Mutex` rather than the `Mutex` itself (and thus avoids the copy restriction on `Mutex`es. Just wondering if there was any architectural reason for not having a pointer receiver?